tests/conditional_compilation/test_utils.py

```diff
@@ -74,7 +74,9 @@ def run_infer(model, out_file, install_dir):
 def make_build(openvino_root_dir, build_dir, install_dir, build_target=None, cmake_additional_args=None, log=None):
     """Parametrized build and install OpenVINO package."""
     additional_args_line = " ".join(cmake_additional_args) + " " if cmake_additional_args else ""
-    build_target_arg_line = f"cmake --build {build_dir} --target {' '.join(build_target)} && " if build_target else ""
+    sea_itt_lib_path = os.path.join(openvino_root_dir, "build_instrumented", "thirdparty", "itt_collector",
```

Comment:
Please use `pathlib` instead `os.path.join` 
why are you use `openvino_root_dir, "build_instrumented"`?

Reply:
I think need to add `target_path` argument